components/eam/bld/build-namelist

```diff
@@ -1705,13 +1705,15 @@ if ( !($phys eq 'ideal' or $phys eq 'adiabatic') ) {
 # Cloud optics; for MMF be specific about which optics package to use
 if ($rad_pkg eq 'rrtmg' or $rad_pkg eq 'rrtmgp') {
     if ($cfg->get('use_MMF')) {
-        if ($cfg->get('MMF_microphysics_scheme') eq 'sam1mom') {
-            add_default($nl, 'liqcldoptics', 'val' => 'slingo');
-            add_default($nl, 'icecldoptics', 'val' => 'ebertcurry');
-        } else {
-            add_default($nl, 'liqcldoptics', 'val' => 'gammadist');
-            add_default($nl, 'icecldoptics', 'val' => 'mitchell');
-        }
+        add_default($nl, 'liqcldoptics', 'val' => 'slingo');
+        add_default($nl, 'icecldoptics', 'val' => 'ebertcurry');
+        # if ($cfg->get('MMF_microphysics_scheme') eq 'sam1mom') {
+        #     add_default($nl, 'liqcldoptics', 'val' => 'slingo');
+        #     add_default($nl, 'icecldoptics', 'val' => 'ebertcurry');
+        # } else {
+        #     add_default($nl, 'liqcldoptics', 'val' => 'gammadist');
+        #     add_default($nl, 'icecldoptics', 'val' => 'mitchell');
+        # }
```

Comment:
yea, I'll do that. I left them there because I assume we'll switch to those optics schemes when @brhillman has a chance to work on this, but it'll be easy enough to add those lines when we get to that point.

Reply:
done.